tests/frontend/schema/class-schema-faq-questions-test.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Frontend\Schema;
+
+use Yoast\WP\Free\Tests\TestCase;
+use WPSEO_Schema_Context;
+use Mockery;
+use WPSEO_Schema_FAQ_Questions_Double;
+
+/**
+ * Class WPSEO_Schema_FAQ_Questions_Test.
+ *
+ * @group schema
+ *
+ * @package Yoast\Tests\Frontend\Schema
+ */
+class WPSEO_Schema_FAQ_Questions_Test extends TestCase {
+
+	/**
+	 * Test setup.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->context = Mockery::mock( WPSEO_Schema_Context::class )->makePartial();
+
+		$this->context->canonical = 'example.com/';
+
+		$this->instance = new WPSEO_Schema_FAQ_Questions_Double( [], [], $this->context );
+	}
+
+	/**
+	 * Tests the FAQ Questions schema output with a question and answer.
+	 *
+	 * @covers \WPSEO_Schema_FAQ_Questions::generate_question_block
```

Comment:
> Use statements always have to be the full name in PHP, so the initial \ is not needed and recommended against (by PHP itself).

Source: https://github.com/Yoast/wordpress-seo/pull/13175#pullrequestreview-257403398

This is also the case for `@covers` AFAIK.
Please check the others in this file too.

```suggestion
	 * @covers WPSEO_Schema_FAQ_Questions::generate_question_block
```